launchpod/angular-launchpod/backend/src/test/java/com/google/launchpod/FormHandlerServletTest.java

```diff
@@ -299,13 +313,17 @@ public void doGet_SingleEntity_ReturnsCorrectXmlString() throws IOException {
   @Test
   public void doGet_MultipleEntities_ReturnsCorrectXmlString() throws IOException {
     DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
-    Entity entity = makeEntity(TEST_PODCAST_TITLE, TEST_MP3_LINK, TEST_XML_STRING);
-    Entity entityTwo = makeEntity(TEST_PODCAST_TITLE, TEST_MP3_LINK, TEST_XML_STRING);
-    String id = KeyFactory.keyToString(entity.getKey());
-    String idTwo = KeyFactory.keyToString(entityTwo.getKey());
+    RSS rss = new RSS(TEST_PODCAST_TITLE, TEST_MP3_LINK);
+    String test_xml_string = RSS.toXmlString(rss);
+
+    Entity entity = makeEntity(TEST_PODCAST_TITLE, TEST_MP3_LINK, test_xml_string);
+    Entity entityTwo = makeEntity(TEST_PODCAST_TITLE, TEST_MP3_LINK, test_xml_string);
     ds.put(entity);
     ds.put(entityTwo);
 
+    String id = KeyFactory.keyToString(entity.getKey());
+    String idTwo = KeyFactory.keyToString(entityTwo.getKey());
```

Comment:
it seems like this method only tests one entity for the correct string, idTwo is not being used

Reply:
I wanted to test the scenario of having multiple entities in Datastore and if doGet() is able to retrieve the correct entity based on entityId. We can talk about removing this test if you think it's unnecessary!